### ISSUE TITLE
docs: `sso.update_profile_information` also supports synchronising the `picture` (avatar) claim

### ISSUE THREAD

**Issue by micolous** [element-hq/synapse#17836]
https://github.com/matrix-org/synapse/pull/13917 added support for synchronising the user's avatar based on an OIDC `picture` claim.

It appears that this can be automatically synchronised on every login when `sso.update_profile_information = true`:

https://github.com/element-hq/synapse/blob/2ce7a1edf7291cfda14bb975135eb7c989d910bc/synapse/handlers/sso.py#L515-L531

However, [the `update_profile_information` docs](https://element-hq.github.io/synapse/latest/usage/configuration/config_documentation.html#sso) state that only "displayname" is supported:

https://github.com/element-hq/synapse/blob/2ce7a1edf7291cfda14bb975135eb7c989d910bc/docs/usage/configuration/config_documentation.md?plain=1#L3680-L3681

This should mention that avatars are also supported.

**Comment by sandhose** [comment]
Good catch! The profile picture claim import has a few caveats, like the fact that it only really work in non-workerized deployments, and doesn't work with external media repositories, which might be why this has been forgotten in the docs.

Nonetheless, this should be updated, so feel free to open a PR to fix this!

**Comment by BloodyIron** [comment]
So is this getting updated? I'm not in a position to make a PR but I had to dig ultra deep through many github threads just to identify this is like the only way to update user avatars from a central authentication system. Not even LDAP/Active Directory integration seems to be able to do this.

Am I to understand that setting "sso.update_profile_information" to "true" updates each user's avatar each time they login or use a session? (which is kind of the point of SSO, using an existing session) As in changes of the avatar in the SSO provider (SAML2/OIDC/CAS/etc) are reflected next session refresh or something like that?
